database/queries.go

```diff
@@ -25,7 +25,9 @@ func (db *Handler) BeginEndEntityOperations() (*Transaction, error) {
 		return nil, err
 	}
 	// lock the table
-	_, err = tx.Exec("LOCK TABLE endentities_lock IN ACCESS EXCLUSIVE MODE")
+	_, err = tx.Exec(`LOCK TABLE endentities IN EXCLUSIVE MODE;
+	LOCK TABLE endentities_lock IN EXCLUSIVE MODE;
```

Comment:
Locking both tables, because they are both important for this.
But still allowing reads to happen. See [pg docs](https://www.postgresql.org/docs/current/explicit-locking.html)

Reply:
Is there a reason this has to go into the same statement?

Reply:
Honestly I thought about grouping all the SQL statements in this function together just because they all either need to happen together or fail anyway. 

This is mostly a habit from much more data driven projects. Less trips to the database is better, but this isn't exactly a bottleneck for us.

Reply:
Word, in offline convo, we talked about you wanting to wait on dropping the use of endentities_lock because you would want to remove it from the schema immediately but I'm being a jerk and asking that we just do this now and drop the database table in this jira ticket [AUT-374](https://mozilla-hub.atlassian.net/browse/AUT-374)

So, please remove the use of endentities_lock

Reply:
Adjusted to remove `endentiteis_lock`